SimplioOfficial/simplio-app
# fix: Stream has already been listened to fix

How to test:
1. Start application.
2. Sign out.
3. Log in.
4. Enter pin.
5. There should be no more `Stream has already been listened to` error.
